modules/metering-cluster-capacity-examples.adoc

```diff
@@ -43,6 +43,6 @@ spec:
     period: "daily"
 ----
 
-<1> To stay organized, remember to change the name of your Report if you change any of the other values.
+<1> To stay organized, remember to change the `name` of your report if you change any of the other values.
```

Comment:
I think it makes more sense to code bracket the `name` column only in the context of a custom resource, but I still need to wrap my head around this new documentation style.

Reply:
@timflannagan Not sure if this helps, but these are the guidelines were trying to follow now for API object formatting: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting